pkg/git/gitlab/gitlab_client.go

```diff
@@ -154,6 +162,13 @@ func (g *GitlabClient) UndoPaCMergeRequest(repoUrl string, d *gp.MergeRequestDat
 		return "", err
 	}
 
+	baseBranchExists, err := g.branchExist(projectPath, d.BaseBranchName)
```

Comment:
Does it make sense to check it here? If `BaseBranchName` does not exist it would exit at `filesExistInDirectory`, wouldn't it?

Reply:
will move above